examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -755,7 +771,20 @@ impl FrontendNode {
             Err(OakStatus::ErrChannelClosed),
             oak::channel_write(out_channel, &data, &[])
         );
+        expect_eq!(Ok(()), oak::channel_close(out_channel.handle));
 
+        // The same is true even if there are messages pending on the channel.
+        let (out_channel, in_channel) = oak::channel_create().unwrap();
```

Comment:
I think read and write are clearer than in and out, but maybe it's just me.

Reply:
Probably yes, but at this point it feels better to be consistent with all the other examples.